umpost/conversion_driver_esm1p5.py

```diff
@@ -188,25 +263,44 @@ def convert_esm1p5_output_dir(esm1p5_output_dir):
         atm_dir_contents, fields_file_name_pattern
     )
 
-    if len(atm_dir_fields_files) > 0:
-         # Run the conversion
-        convert_fields_file_list(atm_dir_fields_files, nc_write_dir)
-    else:
+    if len(atm_dir_fields_files) == 0:
         warnings.warn(
             f"No files matching pattern '{fields_file_name_pattern}' "
             f"found in {current_atm_output_dir.resolve()}. No files will be "
             "converted to NetCDF."
         )
 
-   
+        return  [], [] # Don't try to run the conversion
+
+    succeeded, failed = convert_fields_file_list(
+        atm_dir_fields_files,
+        nc_write_dir
+    )
+
+    return succeeded, failed
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument(
         "current_output_dir", help="ESM1.5 output directory to be converted", type=str
     )
+    parser.add_argument("--quiet", "-q", action="store_true",
+                        help=(
+                            "Report only final exception type and message for allowed"
+                            "exceptions raised during conversion when flag is included."
+                            "Otherwise report full stack trace."
+                        )
+                        )
     args = parser.parse_args()
 
     current_output_dir = args.current_output_dir
-    convert_esm1p5_output_dir(current_output_dir)
+
+    succeeded, failed = convert_esm1p5_output_dir(current_output_dir)
```

Comment:
There's some variable name shadowing with `succeeded` and `failed`.  Using `successes` and `failures` could work as var names, as they're names vs the style of prepending an underscore to "de-shadow" a var name.

Reply:
Good idea! Just fixed this up